platform-hub-web/src/app/identities/identities-manager.component.js

```diff
@@ -0,0 +1,54 @@
+export const IdentitiesManagerComponent = {
+  template: require('./identities-manager.html'),
+  controller: IdentitiesManagerController
+};
+
+function IdentitiesManagerController($scope, hubApiService, gitHubIdentityService, events, _) {
+  'ngInject';
+
+  const ctrl = this;
+
+  ctrl.supported = [
+    {provider: 'keycloak', title: 'Keycloak (using Office 365)', external: false},
+    {provider: 'github', title: 'GitHub', external: true}
+  ];
+
+  ctrl.identities = {};
+
+  ctrl.connect = connect;
+
+  init();
+
+  function init() {
+    // Listen for changes to the Me profile data
+    $scope.$on(events.api.me.updated, (event, meData) => {
+      processMeData(meData);
+    });
+
+    hubApiService.getMe();
+  }
+
+  function connect(service) {
+    // Currently only support Github
+    if (service !== 'github') {
+      return;
+    }
+
+    gitHubIdentityService
+      .popupFlow()
+      .then(hubApiService.getMe);
+  }
```

Comment:
the above function has signature: `Promise<any> or undefined`. Consider returning a promise all the time:

```js
if (service !== 'github') {
      return Promise.reject('Error');
    }
```